main.tf

```diff
@@ -90,10 +90,21 @@ resource "kubernetes_secret_v1" "cluster" {
 ################################################################################
 # Create App of Apps
 ################################################################################
-resource "kubectl_manifest" "bootstrap" {
+resource "helm_release" "bootstrap" {
   for_each = var.create ? var.argocd_bootstrap_app_of_apps : {}
 
-  yaml_body = each.value
+  name       = each.key
+  namespace  = "argocd"
+  repository = "https://bedag.github.io/helm-charts/"
```

Comment:
I don't know who ones this chart.
Can we copy the helm chart to a local directory and use the helm chart from directory instead of relying on this 3rd chart and also network access to the domain `bedag.github.io`

Reply:
Good point the chart is pretty basic I’ll add a local copy